backend/src/test/resources/tests/query/SECONDARY_ID_MIXED/content.csv

```diff
@@ -1,7 +1,7 @@
-pid,sid,value,datum
-a,f_a1,1,"2014-06-30/2015-06-30"
-a,f_a1,1,"2016-06-30/2016-06-30"
-a,f_a2,1,"2014-06-30/2015-06-30"
-a,,1,"2010-06-30/2010-06-30"
-a,f_a3,1.01,"2014-06-30/2015-06-30"
-b,f_b1,1,"2015-02-03/2015-06-30"
\ No newline at end of file
+pid,sid,value,datum,ignored
+a,f_a1,1,"2014-06-30/2015-06-30","a"
+a,f_a1,1,"2016-06-30/2016-06-30","a"
+a,f_a2,1,"2014-06-30/2015-06-30","a"
+a,,1,"2010-06-30/2010-06-30","a"
+a,f_a3,1.01,"2014-06-30/2015-06-30","a"
+b,f_b1,1,"2015-02-03/2015-06-30","a"
```

Comment:
Ist das richtig, dass hier ignored auftaucht?

Reply:
ja, das brauche ich in einem anderen test und soll nichts machen.